dev/feature/3424dfb.unknown-connection-sqlexecutequery/db.bigquery.1.yml

```diff
@@ -0,0 +1,22 @@
+---
+type: Connection
+data:
+  name: "db.bigquery.1"
+  type: "db.bigquery"
+  description: "db.bigquery connection"
+  environment: "iesi-dev"
+  parameters:
+  - name: "host"
+    value: "https://www.googleapis.com/bigquery/v2"
+  - name: "port"
+    value: "443"
+  - name: "project"
+    value: "iesi-01"
+  - name: "dataset"
+    value: "iesi"
+  - name: "authMode"
+    value: "service"
+  - name: "serviceAccount"
+    value: ""
+  - name: "keyPath"
+    value: ""
```

Comment:
Are these files added to the repo as test or example scripts for documentation?

Reply:
These are the scripts to test the implementation